src/java/org/apache/cassandra/db/DeletionTime.java

```diff
@@ -38,14 +41,15 @@ public class DeletionTime implements Comparable<DeletionTime>, IMeasurableMemory
     /**
      * A special DeletionTime that signifies that there is no top-level (row) tombstone.
      */
-    public static final DeletionTime LIVE = new DeletionTime(Long.MIN_VALUE, Integer.MAX_VALUE);
+    public static final DeletionTime LIVE = new DeletionTime(Long.MIN_VALUE, Long.MAX_VALUE);
 
     public static final Serializer serializer = new Serializer();
+    public static final Serializer legacySerializer = new LegacySerializer();
```

Comment:
This seems unused

Reply:
Right, with the back and forth it was left over. These are what `getSerializer()` should return.